web-client/public/js/update-app.js

```diff
@@ -187,14 +187,14 @@ const synchronizeHideAllWeights = () => {
 // Toggle Weighted Functions
 const enableColorOptimal = () => {
     $(BLACK_EDGES).removeClass(ACTIVE_COLOR_OPTION);
-    $(BLACK_EDGES + ">span").removeClass("glyphicon-ok invisible");
+    $(BLACK_EDGES + ">span").removeClass("glyphicon-ok");
```

Comment:
This was a minor tweak, not directly related to #687 but I figured I would tweak it while I was here. Essentially, the `invisible` class is superfluous. If the `glyphicon` class is by itself, without the specific icon identifier, no icon appears anyway. So there’s no need to do anything with `invisible`.